src/common/EIP712/TypedRequestData.ts

```diff
@@ -57,7 +56,7 @@ export default class TypedRequestData implements EIP712TypedData {
   readonly types: Types
   readonly domain: EIP712Domain
   readonly primaryType: string
-  readonly message: RelayRequest
+  readonly message: any
```

Comment:
mmmhhhh....

Reply:
yes - it ISN'T a relayRequest. it is a member to match the format for signing - and relayRequest is of different structure.

Reply:
Could still use a type declaration, just saying...